resources/views/pages/game/[game]/dev-interest.blade.php

```diff
@@ -1,57 +1,67 @@
 <?php
 
-use function Laravel\Folio\{middleware, name};
+use App\Enums\Permissions;
+use App\Models\Game;
+use App\Platform\Services\GameDevInterestPageService;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
 
 middleware(['auth', 'can:view,game', 'can:develop']);
 name('game.dev-interest');
 
-?>
+render(function (View $view, Game $game, GameDevInterestPageService $pageService) {
+    // TODO use a policy -- lift it up to the Folio page middleware. "can:viewDeveloperInterest,game"?
```

Comment:
I believe `can:develop` should be sufficient, but I can't figure out where that logic lives to validate that assumption. Maybe it should be `can:develop,game` to validate set claim for jr. devs?

Reply:
Took me a while to hunt that one down too. It lives in _app/Providers/AuthServiceProvider.php_:

```php
Gate::define('develop', fn (User $user) => $user->hasAnyRole([
    Role::DEVELOPER,
    Role::ARTIST,
    Role::WRITER,
])
    // TODO remove as soon as permission matrix is in place
    || $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper);
```

Given this implementation, I wasn't too sure.

Reply:
That doesn't look like it would allow Jr. Devs access to their claimed sets. It probably should, but that can be addressed as a separate PR.